### ISSUE TITLE
Cannot install Camlistore 

### ISSUE THREAD

**Issue by radkat** [camlistore/camlistore#606]
When using http://camlistore.org/launch/, get the error "An error occurred while creating your instance: "project ID error for doesntmatter: googleapi: Error 400: Invalid value for project: doesntmatter, invalid"."

**Comment by mpl** [comment]
Hmm I can't try with "doesntmatter" as a project id since you own it, but I believe it is a valid project id, and I can't reproduce what you're seeing with a similar one.
As I don't have control on the code on http://camlistore.org/launch/ and I can't add more debug info there, would you mind if I directed you to a dev version that I would host so we can debug that further?

**Comment by radkat** [comment]
Sorry, just saw this notification. Sure, 'd be happy to help debug.

**Comment by cliveb** [comment]
I also had problems using an existing project id setting up using North
West Harvest sign-in. I couldn't use the project ID, or debug it. So I
deleted the existing project ID, but it wouldn't delete, only queue up for
deletion at a future date, the billing cut off date ( I think).

My feeling the http://camlistore.org/launch usability is friendly, but the
API and GCP console is more raw than it looks. I had an opportunity to give
feedback to a research team improving the console, alas, now under NDA.


--//--

tl;dr

Tried to use old project, but can't, so deleted, but can't
"Old project queued for deletion. Won't be deleted till 5-2-2015"

The deletion of the instance failed. Error: The resource
'projects/thermal-scene-750/zones/us-central1-a/instances/instance-1' is
not ready Dismiss

Create new project for Camlistore on GCP

An error occurred while creating your instance: "project ID error for
dogwood-prism-92618: googleapi: Error 403: Access Not Configured. The API
(Compute Engine API) is not enabled for your project. Please use the Google
Developers Console to update your configuration., accessNotConfigured".﻿

--
clive || http://about.me/cliveboulton

**Comment by mpl** [comment]
@radkat great. Could you then please try from https://granivo.re:4430/launch instead? with the same project ID of course.

**Comment by radkat** [comment]
Done! With the same error: "An error occurred while creating your instance: "project ID error for doesntmatter: failed to get project: googleapi: Error 400: Invalid value for project: doesntmatter, invalid"."

**Comment by mpl** [comment]
yep, ok, I see it in the log too, thanks.
I'll dig deeper in the googleapi code to try and narrow it down.

**Comment by mpl** [comment]
@radkat Ok, I think I see what's going on. You haven't created the doesntmatter project yourself before, have you?
I didn't think of asking earlier because I thought we had made it a requirement for the user to enter a project they owned, but I now see that our page actually says "New or existing Google Project.".

Can you please confirm that I got it right, and that if you use a project ID for one you already own, you don't get that error?

**Comment by radkat** [comment]
@mpl, ah I see. "New or existing Google Project" confused me into thinking you will create a project for me. (Maybe change the wording? "Existing Google project with the following enabled: ")

With an already existing project, installation was successful. Thanks for help!

**Comment by mpl** [comment]
@radkat Yes, I was waiting for you to confirm my theory before proposing a change to the wording, thanks.

**Comment by mpl** [comment]
https://camlistore-review.googlesource.com/4998
